modules/fio-uninstall-console.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * security/file_integrity_operator/fio-uninstalling.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="fio-uninstall-console_{context}"]
+= Uninstall the File Integrity Operator using the web console
+
+To remove the File Integrity Operator, you must first delete the `FileIntegrity` objects in the FIO namespace. After the objects are removed, you can then remove the Operator and its namespace.
+
+.Prerequisites
+
+* Access to an {product-title} cluster that uses an account with `cluster-admin` permissions.
+* The File Integrity Operator is installed.
+
+.Procedure
+
+To remove the File Integrity Operator by using the {product-title} web console:
+
+. Navigate to the *Operators* -> *Installed Operators* -> File Integrity Operator page.
+
+. Use the default option *Show operands in: All namespaces* to list all `FileIntegrity` objects in all namespaces.
+
+. Click the Options menu {kebab} and delete all `FileIntegrity` objects.
```

Comment:
Maybe this is obvious in the console, but is there one command to delete all objects, or do you need to delete one at a time? 
```suggestion
. Click the Options menu {kebab} and delete each `FileIntegrity` object.
```